.eslintrc.json

```diff
@@ -16,18 +16,5 @@
         "emsdk/**/*",
         "repros/**/*"
     ],
-    "rules": {
```

Comment:
Removing rules sense they are style related. Style related issue will be addressed through pritter.